packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -65,6 +66,43 @@ def vip_app(container: marathon.Container, network: marathon.Network, host: str,
         raise AssertionError('Unexpected network: {}'.format(network.value))
 
 
+def pod_vip_app(network: marathon.Network, host: str, vip: str):
+    ''' Creates a single app in docker container in pod
+    '''
+    container_port = 0
+    if network == marathon.Network.USER:
+        container_port = unused_port(marathon.Network.USER)
+    test_uuid = uuid.uuid4().hex
+    app = {
+        'id': '/integration-test-{}'.format(test_uuid),
+        'placement': {'acceptedResourceRoles': ['*', 'slave_public']},
+        'containers': [{
+            'name': 'app-{}'.format(test_uuid),
+            'resources': {'cpus': 0.01, 'mem': 8},
+            'image': {'kind': 'DOCKER', 'id': 'debian:jessie'},
+            'exec': {'command': {
+                'shell': '/opt/mesosphere/bin/dcos-shell python '
+                         '/opt/mesosphere/active/dcos-integration-test/util/python_test_server.py {}'.format(
+                             '$ENDPOINT_TEST' if network == marathon.Network.HOST else container_port)
+            }},
+            'volumeMounts': [{'name': 'opt', 'mountPath': '/opt/mesosphere'}],
+            'endpoints': [{'name': 'test', 'protocol': ['tcp'], 'hostPort': 0}],
+            'environment': {'DCOS_TEST_UUID': test_uuid, 'HOME': '/'}
+        }],
+        'networks': [{'mode': 'host'}],
+        'volumes': [{'name': 'opt', 'host': '/opt/mesosphere'}]
+    }
+    if host is not None:
+        app['placement']['constraints'] = [{'fieldName': 'hostname', 'operator': 'CLUSTER', 'value': host}]
+    if vip is not None:
+        app['containers'][0]['endpoints'][0]['labels'] = {'VIP_0': vip}
+    if network == marathon.Network.USER:
```

Comment:
Can we add a case for BRIDGE mode? Would be useful for enabling pods in bridge mode.

Reply:
added